lib/crypto.c

```diff
@@ -73,10 +73,12 @@ crypto_init(void)
         {
           RAND_load_file(rnd_file, -1);
           randfile_loaded = TRUE;
+          if (RAND_status() < 0)
+            {
+              fprintf(stderr,
+                      "WARNING: a trusted random number source is not available, crypto operations will probably fail. Please set the RANDFILE environment variable.");
```

Comment:
Ah, one last thing. We should add more details to this message, the user has already specified RANDFILE as this point, but it didn't provide enough entropy.

Can you please clarify this message a bit?

Thanks